src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndProjData.cxx

```diff
@@ -738,6 +738,29 @@ add_subset_sensitivity(TargetT& sensitivity, const int subset_num) const
   const int min_segment_num = -this->max_segment_num_to_process;
   const int max_segment_num = this->max_segment_num_to_process;
 
+#ifdef STIR_MPI
+     TargetT* dummy_image_pointer = construct_target_ptr();
```

Comment:
this will be a memory leak. Either user a `shared_ptr`, or can you just used the sensitivity image itself (I guess its values will never be used)